index.html

```diff
@@ -0,0 +1,147 @@
+                        At Career Guiders, we help young people explore different career options with the help of
+                        industry experts. Our mission is to reduce unemployment in Africa by funding talents and
+                        connecting young people with experts in their field of interest. Join us today and start your
+                        journey to success!
+                    </p>
+                    <h3 class="message-time">24/7 Active Services</h3>
+                    <p>@Rwanda Kigali converntion center</p>
+                </div>
+            </section>
+            <br><br><br>
+        </section>
+        <section class="main-program" id="main-program">
+            <br>
+            <h4 class="text-center section-title-white">Main Program</h4>
+            <hr class="title-line">
+            <div class="program-holder container">
+
+
+            </div>
+            <br>
+            <div class="container text-center">
+                <a href="#" class="join-btn p-3" id="join">Join us now for support</a>
+                <a href="#" class="program-link p-3">See The whole program</a>
+            </div>
+            <br><br>
+        </section>
+
+        <section class="main-speaker">
+            <br>
+            <h4 class="text-center section-title-black">Featured Speakers</h4>
+            <hr class="title-line">
+            <div class="container">
+
+                <div class="speaker-holder row"></div>
+            </div>
+
+            <div class="more-btn rounded">
+                <p class="text-center pt-2">More<i class="fa-sharp fa-solid fa-chevron-down more-icon"></i></p>
+            </div>
+            <br><br>
+        </section>
+        <section class="partners bg-secondary" id="sponsor">
+            <div class="container">
+                <br>
+                <h4 class="text-center section-title-white">Partner</h4>
+                <hr class="title-line">
+                <div class="container partners-div text-center">
+                    <div class="partner-grp">
+                        <h4 class="partner1">Microverse</h4>
+                        <h4 class="partner2 px-3">Google</h4>
+                        <h4 class="partner3">AWS</h4>
+                    </div>
+                    <div class="partner-grp">
+                        <h4 class="partner4 px-2">IBM</h4>
+                        <h4 class="partner5 px-2">YouTube</h4>
+                    </div>
+                </div>
+            </div>
+            <br><br>
+        </section>
+    </main>
+
+    <footer class="bg-light" id="news">
+        <div class="container footer-section">
+            <img src="./images/aspire-logo.png" width="30%" alt="Aspire log">
+            <div class="pl-5 footer-text"><br> 2023 Aspire Kigali Rwanda <br>
+                <i class="closing-text">Thank you for choosing the rght place for your future</i>
+            </div>
```

Comment:
Dear @Arsenethierry, Also here, as suggested earlier. 
- [ ]  You should stick to the design as much as possible please make the color, spacing, and font similar to the original design and also use the same color for the footer.

### Original design 
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/69077061/534e9c1b-524e-4f77-aa9c-d462efe4f1f6)
### Your design 
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/69077061/a4c9c067-c7df-45f3-a93a-d3c0025e2184)